docs/installation-and-operation/information-collection.md

```diff
@@ -41,6 +41,29 @@ If you're self-hosting Metabase, you can opt out of providing us with your anony
 
 If you're in the process of setting up your Metabase, you can also toggle off tracking during the `Usage Data Preferences` onboarding step. We collect a few anonymous events before that point, but won't do so anymore if you choose to opt out.
 
+## Token validation requests for paid plans
+
+If you're on a paid plan (Pro, Enterprise, or a trial), Metabase periodically sends a token validation request to verify your license. This is separate from the anonymous usage data described above.
+
+### Data included in token validation requests
+
+The token validation request includes:
+
+- Active user count (used for billing)
```

Comment:
Can we explicitly mention somewhere that we don't send any actual data over the network to a random server, and it's all just counts?

And/or maybe even rephrase "Active user count" to "Count of active users", "Count of external users" etc. Uglier but parses better on skimming imo.